bk-nodes/test_bk_nodes.py

```diff
@@ -0,0 +1,66 @@
+import bk_nodes
+import datetime
+import tempfile
+import pytest
+import os
+
+import json
+
+
+# from https://flask.palletsprojects.com/en/1.1.x/testing/
+@pytest.fixture
+def client():
+    app = bk_nodes.app
+    db_fd, app.config['DATABASE'] = tempfile.mkstemp()
+    app.config['TESTING'] = True
+
+    with app.test_client() as client:
+        #with app.app_context():
+        #    init_db()
+        yield client
+
+    os.close(db_fd)
+    os.unlink(app.config['DATABASE'])
+
+
+def test_root(client):
+    rv = client.get('/')
+    assert b'SAGE Beekeeper' in rv.data
+
+
+def test_log_insert(client):
```

Comment:
Same as above, I tested the pagination manually. I could make the page with (size parameter) configurable and include that in the tests.

Reply:
go ahead and think about it...

i figured if it was easy to write a for loop to create a fake 100 entries to ensure you did pagination in ure test it would just improve the code coverage.  because after a short while the table will have atleast 100 entries in it.

Reply:
done